projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/cell.py

```diff
@@ -17,7 +17,7 @@ class Cell:
     def __init__(self, jupyter_cell):
         self.tags = jupyter_cell["metadata"].get("tags", {})
         self.source, self.source_type = self.__extract_source_code(jupyter_cell)
-        self.id = jupyter_cell["id"]
+        self.id = jupyter_cell.get("id")
```

Comment:
This field isn't used for regular notebooks, right? I'm just trying to understand the implications if it's set to `None`

Reply:
Back when we did the investigation, the client data job we tested with had an .ipynb file which was causing the error. We couldn't figure out why that specific notebook did not have an id.